File: week_12/day_1/inheritance/inheritance_start_point/src/main/java/Student.java
public class Student {
    private String name;
    private String cohort;

    public Student(String name, String cohort) {
        this.name = name;
        this.cohort = cohort;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCohort() {
        return cohort;
    }

    public void setCohort(String cohort) {
        this.cohort = cohort;
    }

    public String talk(String language){
        return "I love " + language;
    }
}

File: week_02/day_1/lab_hw_classes/lab_hw_classes_end/student_class/student.rb
class Student

  attr_accessor :name, :cohort

  def initialize(name, cohort)
    @name = name
    @cohort = cohort
  end

  def talk()
    return "I can talk"
  end

  def say_favourite_language(language)
    return "I love #{language}"
  end
end
